app/views/exchanges/hbx_profiles/user_account_index_datatable.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= render "ui-components/v1/navs/primary_nav" %>
+<div class="container">
```

Comment:
having the container class will add horizontal padding, do we want that?

Reply:
I don't have a problem with removing the padding: I just added it to make entries easier to read. I'll take out the padding

Reply:
So I just checked a few of the other tables --> they all appear to all be wrapped in containers prior to rendering the data table. We can change this, but we'd have to change it for all of the admin tables.